sda-svc/values.yaml

```diff
@@ -3,6 +3,9 @@
 # Declare variables to be passed into your templates.
 global:
 
+# Image registry if an private registry is used
+  imageRegistry: ""
```

Comment:
Should this be `image.registry` ?

Reply:
to be consistent sure.

Reply:
I might be mistaken, but with a quick glance in the template files `image.registry` was used, and this might have to match it